docs/index.md

```diff
@@ -3,19 +3,19 @@
   title: ""
 ---
 
-# AddressBook Level-3
+# UGTeach
 
 [![CI Status](https://github.com/se-edu/addressbook-level3/workflows/Java%20CI/badge.svg)](https://github.com/se-edu/addressbook-level3/actions)
 [![codecov](https://codecov.io/gh/se-edu/addressbook-level3/branch/master/graph/badge.svg)](https://codecov.io/gh/se-edu/addressbook-level3)
```

Comment:
Can help change URLs to this:
1st: https://codecov.io/gh/AY2425S1-CS2103T-F14a-1/tp/graph/badge.svg?token=XH80L1OD62
2nd: https://codecov.io/gh/AY2425S1-CS2103T-F14a-1/tp

Reply:
Thanks for catching this! Ive made the changes.

Reply:
I think you forgot to change the svg link